dev/create-release/release-build.sh

```diff
@@ -210,6 +210,8 @@ if [[ "$1" == "package" ]]; then
     PYSPARK_VERSION=`echo "$SPARK_VERSION" |  sed -e "s/-/./" -e "s/SNAPSHOT/dev0/" -e "s/preview/dev/"`
     echo "__version__='$PYSPARK_VERSION'" > python/pyspark/version.py
 
+    export MAVEN_OPTS="-Xmx12000m"
```

Comment:
Can we have this globally outside of `if` statement? Then, it looks like we need only one line addition.

Reply:
okay

Reply:
nit: we can say `-Xmx12g`?

Reply:
ok.